src/routes/auth.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+
+const controllers = require('../controllers/auth');
+const { validator } = require('../middlewares/validation');
+const schema = require('../schemas/auth');
+const { authorize } = require('../middlewares/authorization');
+
+const router = express.Router();
+
+router.post(
+  '/_users',
+  authorize,
```

Comment:
I thought the super user is the only user who can create other users, but we can remove the authorization from this route.

Reply:
Aha, I guessed it. We can decide about it. @IanMayo What's your take on this?

Reply:
I'd like to lock it down as much as possible :-)

So, I think we need to authenticate the role that is adding a new user - and ensure that role is authorised to add new users.  Otherwise someone could use `curl` to create their own account, and start mangling data.

Reply:
That's a good point! Let's leave it to be like this then.